### DISCUSSION TITLE
I want to search streamfield in admin panel?

### DISCUSSION THREAD

**Discussion by envobe** [wagtail/wagtail#8556] [General]
I have about 400 different types of blocks. How can I search for them from the admin panel?

Example image:

[Example search form admin panel](https://i.stack.imgur.com/84YmK.png)

**Comment by Scotchester** [comment]
Are you trying to search for _content_ within a block, or just for where certain block types are used? If it's the latter, check out https://github.com/cfpb/wagtail-inventory.

**Reply by envobe** [reply]
Hello, I have a lot of block space as seen in the ADDITIONAL picture. 400 pcs :)

Since I don't know how to group them, I want to add a cross-block speed dial box.

I don't know how to group these blocks.

It made sense to search and use the block name for quick access.

Thank u.
![1](https://user-images.githubusercontent.com/5060075/168866004-00c39025-3382-483a-a4db-03b898b32216.JPG)

**Comment by Scotchester** [comment]
Ahhh, I see what you're getting at. I must be honest, I'm a little shocked at the number of blocks you have, and wonder if a different approach (like putting options on a single Hero Section block to switch between the 22 different variations).

**Comment by envobe** [comment]
Hi,

I have 400 blocks, not 22. :)

What should I do to get easy access to these blocks? Can I add a search form?

Or can I group? For example;

**All hero section (When I click it, all subsections open.)**
    - Hero section 1
    - Hero section 2

**All features (When I click it, all subsections open.)**
    - Feature 1
    - Feature 2

bla bla...

Thank u.
